src/lib/settings.js

```diff
@@ -1,17 +1,16 @@
 module.exports = {
   setGTM: ({
     app,
-    analyticsCookieDomain,
-    uaContainerId,
-    isGa4Enabled,
     ga4ContainerId,
-    gaTaxonomyLevel2,
+    uaContainerId,
+    analyticsCookieDomain,
+    ga4Disabled,
+    uaDisabled,
   }) => {
-    app.set("APP.GTM.UA_CONTAINER_ID", uaContainerId);
+    app.set("APP.GTM.ID", ga4ContainerId);
```

Comment:
Given we've got other variables prefixed with GA4 or UA, it would be easier to understand which ID is being used if this variable mentioned GA4.

Once we've migrated everyone off UA, we can then refactor this to be a more simple `ID`

Reply:
Updated.